platformio.ini

```
@@ -10,17 +11,49 @@ debug_tool = stlink
 monitor_speed = 115200
 board_build.ldscript = stm32c092rbtx_flash.ld
 
+lib_deps = 
+    https://github.com/eagletrt/libeagletrt-sw.git#v0.1.0
+
 build_src_filter = 
     +<Core/Src/>
+    +<Core/Src/cockpit/>
     +<Drivers/STM32C0xx_HAL_Driver/Src/>
     -<Drivers/STM32C0xx_HAL_Driver/Src/*_template.c>
     +<*.s>
 
 build_flags = 
+    -Ilibs/libeagletrt-sw/include
 	-ICore/Inc
+    -ICore/Inc/cockpit
     -IDrivers/CMSIS/Include
     -IDrivers/STM32C0xx_HAL_Driver/Inc
     -IDrivers/STM32C0xx_HAL_Driver/Inc/Legacy
     -IDrivers/CMSIS/Device/ST/STM32C0xx/Include
     -DUSE_HAL_DRIVER
     -DSTM32C092xx
+
+[env:unit_tests]
+platform = native
+test_framework = unity
+test_build_src = true  ; (Updated from deprecated 'test_build_project_src')
+
+lib_deps = 
+    https://github.com/eagletrt/libeagletrt-sw.git#v0.1.0
+
+build_flags = 
+    -Itest/
+    -ICore/Inc
+    -ICore/Inc/cockpit
+    -Ilibs/libeagletrt-sw/include
+    -D EAGLETRT_STATIC=
```

```suggestion
    -DEAGLETRT_STATIC=
```
To keep it "clean" with the whole file style.